store/src/db/mod.rs

```diff
@@ -49,25 +49,33 @@ impl Db {
             create_dir_all(p)?;
         }
 
-        let pool =
-            SqliteConfig::new(config.database_filepath.clone()).create_pool(Runtime::Tokio1)?;
-
-        let conn = pool.get().await?;
+        let pool = SqliteConfig::new(config.database_filepath.clone())
+            .builder(Runtime::Tokio1)?
+            .post_create(Hook::async_fn(move |conn, _| {
+                Box::pin(async move {
+                    // Feature used to support `IN` and `NOT IN` queries
+                    let _ = conn
+                        .interact(|conn| array::load_module(conn))
+                        .await
+                        .map_err(|_| HookError::StaticMessage("Loading carray module failed"))?;
```

Comment:
I would add a comment here to explain that we need to do this on every connection when they are created.